kvdb/src/io_stats.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Generic statistics for key-value databases
+
+/// Statistic for the `span` period
+pub struct IoStats {
+	/// Number of transaction.
+	pub transactions: u64,
+	/// Number of read operations.
+	pub reads: u64,
+	/// Number of reads resulted in a read from cache.
+	pub cache_reads: u64,
```

Comment:
This is not set anywhere? I suppose we can only get cache stats from the underlying DB, since kvdb itself does not do any caching

Reply:
At the moment, I don't put counting cache reads in stats because it might have performance impact

It probably should be counted only conditionally later